Sources/Templates/Template.c

```diff
@@ -4,7 +4,7 @@
 #include <stdlib.h>
 #include <assert.h>
 
-void Template_AddMinuitia(Template *template, TemplateMinutia *minutia)
```

Comment:
Are we going to use `const` consistently?  I'd have thought both should be `const` in this situation?